warp/service/warp_client.go

```diff
@@ -0,0 +1,51 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/utils/cb58"
+	"github.com/ava-labs/subnet-evm/plugin/evm/message"
+	"github.com/ava-labs/subnet-evm/rpc"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+)
+
+var _ WarpClient = (*warpClient)(nil)
+
+type WarpClient interface {
+	GetSignature(ctx context.Context, signatureRequest message.SignatureRequest) (*hexutil.Bytes, error)
```

Comment:
```suggestion
	GetSignature(ctx context.Context, messageID ids.ID) ([]byte, error)
```

We should not import the message package at all for this.